nodejs/test/layer.tap.js

```diff
@@ -0,0 +1,37 @@
+/*
+ * Copyright 2020 New Relic Corporation. All rights reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+//
+// process.env.NEW_RELIC_LAMBDA_HANDLER = 'myFunction.handler'
+// process.env.NEW_RELIC_APP_NAME = 'My Test App'
+
+const tap = require('tap')
+const { handler, getHandlerPath } = require('../index')
+const { TestAgent } = require('@newrelic/test-utilities')
+
+tap.test('The handler wrapper wraps the handler', async (t) => {
```

Comment:
this reads a little funny. should it be something like "New Relic handler wraps the customer's handler"?